spec/features/signup_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+feature 'allow user to register account' do
+  scenario 'sign up user' do
+    expect { sign_up(email: 'mat@makers.com', username: 'mat') }.to change{User.count}.by(1)
```

Comment:
Metrics/LineLength: Line is too long. [92/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)